### ISSUE TITLE
Send an email to interested parties if a Service team schedules an API Review with not found onboarding record

### ISSUE THREAD

**Issue by maririos** [Azure/azure-sdk-tools#5987]
Related to: https://github.com/Azure/azure-sdk-tools/issues/3918

The email should be sent to:
- Creator of meeting
- cc:  azsdkinfoboard, Mark, and Mike

Content of the email:
> ...

**Comment by JonathanCrd** [comment]
Based on this comment: https://github.com/Azure/azure-sdk-tools/issues/3918#issuecomment-1511767179
> When there is no onboarding record, we should sent a separate email to the creator of the meeting asking them to Schedule an Informational meeting.

Shouldn't we asked them to schedule an Informational meeting first instead directly in the app before they schedule the API Review?
We could show a pop-up or a message once they select the service and product.

**Comment by maririos** [comment]
What we don't want to do is to block the users from scheduling the API Review meeting. So another idea could be that after they schedule the meeting, instead of an email they get the pop-up asking them to schedule an Informational and then redirect them.

What do you think?

**Comment by JonathanCrd** [comment]
I like that option too. However, I'd like to understand what should happen first. @ladonnaq If a team that hasn't onboard yet schedule an API Review meeting, are they expected to have an Informational Meeting before the API Review? Or it doesn't matter in which order they have them?

**Comment by maririos** [comment]
Ideally they should have the Informational meeting first, but because it can take up to a month to get a meeting with the API Review, we don't want to block them. they can schedule and then schedule the informational which has more availability.

FYI @mikekistler as we talked about this a couple of weeks ago

**Comment by JonathanCrd** [comment]
Fair point. Thanks for clarifying.

**Comment by ladonnaq** [comment]
> Ideally they should have the Informational meeting first, but because it can take up to a month to get a meeting with the API Review, we don't want to block them. they can schedule and then schedule the informational which has more availability.
> 
> FYI @mikekistler as we talked about this a couple of weeks ago

+1  This make sense, we don't want to block them but we should explain that the team needs to schedule the onboarding meeting before the API Review. Otherwise, there is often confusion as the teams are not prepared for the API Review.  (Because they did not onboard and do not understand our end-to-end process).

**Comment by maririos** [comment]
@ladonnaq will this be a problem for services that are not in scope for APEX? for example, if Storage tries to schedule a meeting, and we don't find an Onboarding issue, we will ask them to meet with you even though they have been working with us for a while.
How will that work?

**Comment by maririos** [comment]
@ladonnaq and @JonathanCrd the more I think about this, the more I believe the teams should talk to us if they don't have an Onboarding WI with us.
This will allow us to:
- Verify data
- Open one in case it was non apex and therefore they never had one
- Learn about them

Because of that, I think we could do 2 things:
- Take the responsability and prompt to them that we need more information to continue (withouth blocking them) send an email to our team with the information about their meeting and for us to follow up with them and see what is happening
- Ask them to schedule an Informational meeting and get the information there

Open to any suggestions

**Comment by JonathanCrd** [comment]
Agree, I think we can improve the flow here to guide the user to schedule an informational meeting after they schedule the API Review.

Actually, we will know if they need this before they even click the "Create" button. We could prompt them to schedule an Informational at that moment.

**Comment by maririos** [comment]
The question is, do we need a meeting with them or can we do it offline over email to make sure we don't create another road block for services to move on the process.
I am leaning more towards option 1) of informing them that someone from our team will contact them and then sending us an email about the team, we try to figure out what information we have about them, and we contact them or ask them for a meeting if we need more information.
@ladonnaq what do u think?

**Comment by maririos** [comment]
Ping @ladonnaq  :)

**Comment by maririos** [comment]
@ladonnaq would you be ok if we implement this after the dashboard work is done?

**Comment by maririos** [comment]
Instead of this approach, it will be better to relate the meetings with a Release Plan https://github.com/Azure/azure-sdk-tools/issues/6819. If there is no release plan, then the user should create one. This will also force them to have an informational meeting if they haven't yet.
Because of that, I am going to close this issue.
